src/site/content/en/learn/css/css.11tydata.js

```diff
@@ -5,11 +5,22 @@ module.exports = function () {
     // e.g. A course with a key of 'a11y' would have a corresponding
     // _data/courses/a11y directory.
     projectKey: 'css',
+    searchTag: 'course-learn-css',
```

Comment:
I think this should go in meta.yml as well. Ideally the only value in here that's project specific would be the projectKey and I'm even wondering if we should just infer that from the directory name (that's an issue for a different PR tho 😛 ). But I'd like to remove project specific stuff so authors don't forget to configure it.

Reply:
I don't want this to be editable by course authors, as it has nothing to do with the content of the course and is used just to technically adjust our search infra. But you're right, it can also be simply derived from the projectkey and not editable at all.